add task solution

nikassafronovs opened the pull request.

nikassafronovs committed:
add task solution

2pasha reviewed (changes requested):
almost good, great job!

please fix these points:

1. please add DEMO LINK 

2. when user hover on this icon - phone number should be appeared
<img width="683" height="209" alt="image" src="https://github.com/user-attachments/assets/71df6a1d-f647-4f84-9827-74cf6f7903c3" />

3. add hover effects for images
<img width="740" height="497" alt="image" src="https://github.com/user-attachments/assets/ce486dd1-5e99-422c-9cae-fc588d2d9b95" />

4. for this images make hover scale little bit smaller
<img width="738" height="389" alt="image" src="https://github.com/user-attachments/assets/f3276ba3-6f78-483c-8751-e56c6193863f" />

5. add some left paddings and add placeholder for textarea
<img width="424" height="257" alt="image" src="https://github.com/user-attachments/assets/903eeb99-c7e1-4956-88d2-be9720ace72f" />

6. disable resizing for textarea 
<img width="312" height="227" alt="image" src="https://github.com/user-attachments/assets/022b9765-08d3-47ee-ad59-47340429cff2" />

7. add submit button to form (which also must clear the form's inputs)

8. change color for this line
<img width="242" height="379" alt="image" src="https://github.com/user-attachments/assets/d3f672d6-c208-4d61-b68c-65114743d562" />

9. this text on tablet must be in two lines
<img width="384" height="450" alt="image" src="https://github.com/user-attachments/assets/7646c141-9143-4428-ab8f-843b9bd8097c" />

10. for mobiles this section has different layout
<img width="611" height="969" alt="image" src="https://github.com/user-attachments/assets/3df20575-e4ee-4b62-96c6-1cfdee9b8bc9" />

nikassafronovs committed:
fixed problems

nikassafronovs commented:
Hello, I fixed problems abd9401c96fdd77af47c3edccd8f44fbba873815

And I add demo link - [DEMO LINK](https://nikassafronovs.github.io/layout_landing-page/)

brespect reviewed (changes requested):
Good progress you did! Let`s check next:
1. There should be the info about phones when hovering on header phone icon:
<img width="165" height="65" alt="Знімок екрана 2025-11-27 о 13 20 55" src="https://github.com/user-attachments/assets/b63410a8-c390-4946-8a97-3ff21c763c72" />

2. Contacts section looks different from the mockup:
<img width="733" height="431" alt="Знімок екрана 2025-11-27 о 13 22 33" src="https://github.com/user-attachments/assets/f7c38fba-4d6a-4e34-ab97-c58ca980ddba" />

3. Add some top padding for the textaream message field, now it looks a bit strange: 
<img width="498" height="165" alt="Знімок екрана 2025-11-27 о 13 23 02" src="https://github.com/user-attachments/assets/e05be83f-8a7b-4714-b216-3be306987274" />

4. Pictures look broken on Mobile version, check this:
<img width="211" height="556" alt="Знімок екрана 2025-11-27 о 13 26 07" src="https://github.com/user-attachments/assets/446735fd-b4cb-47c3-8d6d-58bd93e35252" />

nikassafronovs committed:
compleat task

nikassafronovs commented:
Corrected the comment 

002e3ee00e08c51b357826335c2bb98978a9761a

brespect reviewed (changes requested):
Almost done!
1. There should be the gap between phone icon and text info:
<img width="430" height="141" alt="{3F2DB482-E65A-4CF0-9121-8E53474C3723}" src="https://github.com/user-attachments/assets/9baa148f-13d5-49d6-81a3-568d3b474961" />

nikassafronovs committed:
fixed gap phonetip

nikassafronovs commented:
Fixed gap phonetip 

4c109038fc61fa07cc4cdfd12342b8a7b4731c7b

FaiHamid reviewed (changes requested):
Good job!
Let`s fix a few details:

1. Remove scroll while menu open
<img width="517" height="918" alt="image" src="https://github.com/user-attachments/assets/6f011387-6ecb-444a-b881-1c3befd4cf72" />

2. Reduce hover for rectangle images
<img width="1446" height="528" alt="image" src="https://github.com/user-attachments/assets/a60db30f-81ff-43c8-8b7b-cb2cf82b733d" />

3. After submiting page shouldn`t refresh
<img width="589" height="502" alt="image" src="https://github.com/user-attachments/assets/2e0a4eee-2d11-48eb-8a9a-0b81aff53cd7" />

nikassafronovs committed:
fixed menu, reduce hover imges, fixed button

nikassafronovs commented:
I don't know how to remove scrolling when the menu is open, ChatGPT says to do it in JavaScript, but I haven't gone through it yet. So I removed the extra scrolling elements that appeared.

I reduced hover for rectangle images and changed type on button, and now page doesn't refresh.

d53c74c8f4c74b0528e865f798b4818adb982ded

2pasha reviewed (approved):
great job! 🔥

1. you may try to use this code to disable scroll, just paste it into your main script file
```
function handleScrollLock() {
  const currentHash = window.location.hash;
  const body = document.body;

  if (currentHash === '#menu') {
    body.style.overflow = 'hidden';
  } else {
    body.style.overflow = '';
  }
}

handleScrollLock();

window.addEventListener('hashchange', handleScrollLock);
```
also i strongly recommend you to paste this code into your GPT and ask it to explain this code for you

2. also please clear all inputs in the from after user click on submit button

nikassafronovs committed:
add function on js
